### DISCUSSION TITLE
Installation issue

### DISCUSSION THREAD

**Discussion by Rakan256** [CompuCell3D/CompuCell3D#772] [Q&A] [answered]
Hello,
I have tried downloading and installing the app compucell3d and followed the manual exactly, i tried 4.6.0 and 4.5.0 and I keep getting the same issue. When I try to open the app the command window opens then immediately closes afterwards. What might be the issue how can I fix this?

**Comment by jpsluka** [comment]
Are you comfortable doing a command line install using conda/minconda? If so, can you try the instructions at
(https://compucell3d.org/SrcBin#Conda)

**Comment by maciekswat** [comment, marked as answer]
Indeed, I can confirm these issues. They all originate from the fact that the Mamba version has been changed upstream and some changes in Mamba make it throw an error regardless of how you install CC3D I have built the new installer and it seems to be working fine. It also has a better logger that will catch all information from the installation even if it  fails
Remove old installer and re-download the new installer 
https://sourceforge.net/projects/cc3d/files/4.6.0/windows/CompuCell3D-setup-4.6.0.0.exe/download

If you encounter any issues please let us know

**Comment by maciekswat** [comment]
To follow up on this - when you try to install cc3d using mamba - following instructions that @jpsluka mentioned you will still get an error:
```commandline
mamba install -c main -c conda-forge -c compucell3d  compucell3d=4.6.0
main/noarch                                        547.6kB @ 594.5kB/s  0.6s
main/win-64                                          4.0MB @   1.1MB/s  3.5s
conda-forge/noarch                                  17.3MB @   4.0MB/s  4.3s
conda-forge/win-64                                  25.6MB @   5.2MB/s  4.8s
critical libmamba [json.exception.parse_error.101] parse error at line 163, column 37: syntax error while parsing value'
```

This is the bug that Mamba developers introduced somewhere. and it affects Windows users who run Mamba using a "regular terminal"  - ``%windir%\system32\cmd.exe`` Here is the fix:

Prior to running cc3d installation command run this in your terminal (again this happens only on windows):

```commandline
set PYTHONIOENCODING=utf-8
``` 
and then run cc3d install command:

```commandline
mamba install -c main -c conda-forge -c compucell3d  compucell3d=4.6.0
```
